src/cdk-experimental/listbox/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("//tools:defaults.bzl", "ng_module", "ng_test_library", "ng_web_test_suite", "sass_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+ng_module(
+    name = "listbox",
+    srcs = glob(
+        ["**/*.ts"],
+        exclude = ["**/*.spec.ts"],
+    ),
+    module_name = "@angular/cdk-experimental/listbox",
+    deps = [
+        "//src/cdk/a11y",
+        "//src/cdk/bidi",
+        "//src/cdk/keycodes",
+        "//src/cdk/overlay",
+        "//src/cdk/portal",
+        "@npm//@angular/animations",
+        "@npm//@angular/core",
+        "@npm//rxjs",
```

Comment:
I'm not sure if this file has been built up from an existing prototype you already have, but generally I'd prefer us adding these dependencies on-demand. Otherwise we might end up with unused dependencies that slow-down CI and local development (same for the loaded, but unused rules above)

Reply:
Ah okay that makes sense. Yeah I'll just remove it and then add them as needed.